arch/arm/src/stm32f7/stm32_qspi.c

```diff
@@ -1939,7 +1939,7 @@ static void qspi_setmode(struct qspi_dev_s *dev, enum qspi_mode_e mode)
         }
 
       qspi_putreg(priv, regval, STM32_QUADSPI_DCR_OFFSET);
-      spiinfo("DCR=%08x\n", regval);
+      spiinfo("DCR=%08lx\n", regval);
```

Comment:
BTW, shouldn't this be `spiinfo("DCR=%08" PRIx32" \n", regval);`?

Reply:
Done